resources/views/Checkout/Confirmation.twig

```diff
@@ -0,0 +1,17 @@
+<script>
+    {% if success == "true" %}
+    OffAmazonPayments.initConfirmationFlow("{{ sellerId }}", "{{ amazonReferenceId }}", function (confirmationFlow) {
+        confirmationFlow.success();
+    });
+    {% else %}
+    OffAmazonPayments.initConfirmationFlow("{{ sellerId }}", "{{ amazonReferenceId }}", function (confirmationFlow) {
+        confirmationFlow.error();
+    });
+    {% endif %}
+</script>
+{% if config('Payone.PAYONE_PAYONE_AMAZON_PAY.sandbox') == "true" %}
```

Comment:
Das Script ist doch bereits initial geladen auf der Seite des Checkouts?
Hier ist doch der Teil, der in dem Overlay angezeigt wird. Da diese innerhalb der Seite ist, ist auch das Script verfügbar.